Structure_Engine/Create/Constraints/Constraint6DOF.cs

```diff
@@ -106,6 +118,15 @@ public static Constraint6DOF FullReleaseConstraint6DOF(string name = "Release")
 
         /***************************************************/
 
+        [Description("Creates a Constraint6DOF from a set of booleans. True means fixed, false means free.")]
```

Comment:
denotes free

I think we should keep consistent language with the LinkConstraint